web/modules/custom/dpl_react_apps/src/Controller/DplReactAppsController.php

```diff
@@ -135,7 +135,7 @@ public function search(): array {
       'results-text' => $this->t('results', [], ['context' => 'Search Result']),
       'show-more-text' => $this->t('Show more', [], ['context' => 'Search Result']),
       'show-results-text' => $this->t('Show results', [], ['context' => 'Search Result']),
-      'showing-results-for-text' => $this->t('Showing results for', [], ['context' => 'Search Result'])->__toString(),
+      'showing-results-for-text' => $this->t('Showing results for "@query"', [], ['context' => 'Search Result'])->__toString(),
```

Comment:
Consider if we really need to call `__toString()` here.

We do not do so for other texts here. I see that we need it for the handling in `novel_preprocess_dpl_react_app__search_result()`. An alternate approach might be to move all the handling there á la:

```php
$loading_text = $variables['data']['showing-results-for-text'] ? (string) $variables['data']['showing-results-for-text'] : "";
```